testcases/smbtorture/test_smbtorture.py

```diff
@@ -97,9 +127,9 @@ def generate_smbtorture_tests(test_info_file):
 )
 def test_smbtorture(share_name, test):
     ret = smbtorture(share_name, test, output)
+    if os.path.exists(output):
+        os.unlink(output)
```

Comment:
This is potentially racy (Look before you leap). I think it is preferable to do a:
```suggestion
try:
    os.unlink(output)
except FileNotFoundError:
    pass
````

But this is very minor, if you don't feel like doing this I don't mind.

Reply:
We are unlikely to hit a race here since only one instance of the runner is expected to call smbtorture. In any case, your suggestion is a good one which can be implemented as a helper function since I've had to implement simple check and unlink for temporary files elsewhere. For now I'll skip this and create a helper to handle deletion of temporary files in another commit.

Reply:
Sure. I also just realized you could also use pathlib here if you want to and the Path.unlink object has a `missing_ok` flag: https://docs.python.org/3/library/pathlib.html#pathlib.Path.unlink